### PR TITLE
fix link update user page with backend

### PR DESCRIPTION
**Pull request by mes1901**
https://trello.com/c/crh2qVQA

### CODE REVIEW

File: backend/tests/Feature/UserAuthTest.php [hunk 1]
```diff
@@ -47,4 +49,21 @@ public function testFailLogin()
...
+        $headers = ['Authorization' => "Bearer $token"];
+
+        $response = $this->actingAs($user)->put('api/v1/users/me', [
+            'name' => 'Test',
+            'email' => 'test@unique.com',
+            'password' => ''
+        ], $headers);
+
+//        dd($response->json());
+
```
Comment (lenchvolodymyr) [inline backend/tests/Feature/UserAuthTest.php]: remove comment

File: backend/tests/Feature/UserAuthTest.php [hunk 2]
```diff
@@ -47,4 +49,21 @@ public function testFailLogin()
...
+        $this->assertEquals(200, $response->getStatusCode());
```
Comment (lenchvolodymyr) [inline backend/tests/Feature/UserAuthTest.php]: also check database that user has changed. https://laravel.com/docs/5.8/database-testing#available-assertions

File: frontend/src/components/auth/UpdateProfileForm.vue [hunk 3]
```diff
@@ -138,12 +139,19 @@
...
+
             onSave() {
                 if (this.$refs.form.validate()) {
-                    this.update(this.editUser)
-                        .catch((error) => {
-                            alert(error.message);
-                        });
+                    this.update(this.editUser).then(() => {
+                        this.$emit("success");
+                        this.showSuccessMessage('Profile have been successfully updated!');
```
Comment (lenchvolodymyr) [inline frontend/src/components/auth/UpdateProfileForm.vue]: Profile **has** been ...

**Comment by TatsuUkraine** [comment]
@mes1901 why you need changed lock file?

**Comment by mes1901** [comment]
> @mes1901 why you need changed lock file?

There were problems launching the backend. Therefore, I had to do a complete rebuild of the project. Maybe because of this?

**Comment by TatsuUkraine** [comment]
Could be, in any case - revert lock file, and try to don't commit it next time, unless you installed new deps. Also - if you need to just install deps, better use `npm ci` next time, it's more secured

**Comment by TatsuUkraine** [comment]
@mes1901 please fix travis issues, and also it seems you have changes in Docker config (I'm not sure that it's right), @lenchv ?

File: .aws/nginx/Dockerfile [hunk 4]
```diff
@@ -2,3 +2,4 @@ FROM nginx:1.17.2-alpine
 
 COPY .aws/nginx/.certs/ /nginx/certs
 COPY .aws/nginx/nginx.conf /etc/nginx/conf.d/default.conf
+COPY .aws/nginx/pki/C199461B0078D4FC6F31B9B0282DAB60.txt /pki/.well-known/pki-validation/file.txt
```
Comment (lenchvolodymyr) [inline .aws/nginx/Dockerfile]: get rid of this

File: .aws/nginx/nginx.conf [hunk 5]
```diff
@@ -52,6 +52,11 @@ server {
         autoindex off;
     }
 
+    location ^~ /.well-known/pki-validation {
```
Comment (lenchvolodymyr) [inline .aws/nginx/nginx.conf]: remove this

File: .aws/nginx/pki/C199461B0078D4FC6F31B9B0282DAB60.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+175937670D1CAA346914903D336C9597921C543A9F53AFD9DFFB9FBEA71E5A10 comodoca.com 5d51b5fd6f09c
```
Comment (lenchvolodymyr) [inline .aws/nginx/pki/C199461B0078D4FC6F31B9B0282DAB60.txt]: remove this

**Pull request merged**